recipes/libyuv/all/conandata.yml

```
@@ -0,0 +1,8 @@
+sources:
+  "20201106":
+    url: "https://chromium.googlesource.com/libyuv/libyuv/+archive/2664f649bc0a32e05f5bc3ff4d12134e41aa08c4.tar.gz"
+
+patches:
+  "20201106":
```

```suggestion
  "cci.20201106":
```

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

---

fixed